openpype/settings/entities/schemas/projects_schema/schemas/schema_anatomy_attributes.json

```
@@ -69,6 +69,11 @@
             "type": "tools-enum",
             "key": "tools_env",
             "label": "Tools"
+        },
+        {
+            "type": "boolean",
+            "key": "active",
+            "label": "Is Project Active"
```

```suggestion
            "label": "Active Project"
```

The only suggestion I have is to rename this label to a noun rather than a question. Other than that. All good